backend/config.ts

```diff
@@ -58,7 +58,7 @@ export const apis = {
     url: process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
     ui_url: process.env.OAUTH_ENDPOINT_UI_URL || process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
     timeoutSeconds: toNumber(process.env.API_ENDPOINT_TIMEOUT_SECONDS) || 10,
-    clientId: process.env.API_CLIENT_ID || 'elite2apiclient',
+    clientId: process.env.API_CLIENT_ID || 'prisonapiclient',
```

Comment:
We now use the the prisonapiclient in local development as that that is backed by the correct callback urls in the auth app